projects/rero/ng-core/src/lib/record/editor/editor.component.html

```diff
@@ -14,67 +14,101 @@
+        </span>
+      </legend>
+      <!-- buttons -->
+      <div class="actions">
+        <!-- cancel button -->
+        <button type="button"
+                id="editor-delete-button"
+                class="btn btn-outline-danger btn-sm"
+                (click)="cancel()">
           <i class="fa fa-times"></i>
           {{ 'Cancel' | translate }}
         </button>
-        <button
-          id="editor-save-button"
-          type="submit"
-          class="btn btn-primary btn-sm"
-          [disabled]="!form.valid"
-        >
-          <i class="fa fa-save"></i>
-          {{ 'Save' | translate }}
+        <!-- load from template button -->
+        <button *ngIf="editorSettings.template.loadFromTemplate"
+                type="button"
+                id="editor-load-template-button"
+                class="btn btn-outline-primary btn-sm ml-1"
+                (click)="showLoadTemplateDialog()">
+          <i class="fa fa-upload"></i>
+          {{ 'Load template' | translate }} …
         </button>
+        <!-- save button -->
+        <div class="btn-group" dropdown>
+          <button type="submit"
+                  id="editor-save-button"
+                  class="btn btn-primary btn-sm ml-1"
+                  [disabled]="!form.valid">
+            <i class="fa fa-save"></i>
+            {{ 'Save' | translate }}
+          </button>
+          <ng-container *ngIf="saveAlternatives.length > 0">
+            <button type="button"
+                    id="editor-save-button-split"
+                    dropdownToggle
+                    class="btn btn-sm btn-primary dropdown-toggle-split"
+                    aria-controls="dropdown-split">
+              <i class="fa fa-caret-down"></i>
+            </button>
+            <ul id="editor-save-button-dropdown-split"
+                *dropdownMenu
+                class="dropdown-menu dropdown-menu-right"
+                role="menu"
+                aria-labelledby="button-split">
+              <li role="menuitem" *ngFor="let entry of saveAlternatives">
+                <a class="dropdown-item" (click)="entry.action(entry, this)">{{ entry.label }}</a>
+              </li>
+            </ul>
+          </ng-container>
+        </div>
       </div>
-    </form>
+    </div>
+
+
+    <!-- add fields section -->
+    <div *ngIf="editorSettings.longMode" class="col-md-3 col-xl-2 bd-sidebar">
+      <ng-core-editor-add-field-editor
+        class="bd-search d-flex align-items-center"
+      >
+      </ng-core-editor-add-field-editor>
+    </div>
+
+    <!-- TOC navigation -->
+    <nav
+      *ngIf="editorSettings.longMode"
+      class="d-none d-xl-block col-xl-2 bd-toc"
+      aria-label="Secondary navigation"
+    >
+      <label for="addField" translate>Jump to</label>
+      <ul class="section-nav">
+        <li *ngFor="let f of tocFields" class="toc-entry toc-h2">
+          <a href (click)="setFocus($event, f)">{{ f.templateOptions.label }}</a>
```

Comment:
`href` ?

Reply:
without this property, browser consider this link as a non-link and we can't display it as a link.
This code came from previous PR, I just change the position of this code in the template ;-)